### ISSUE TITLE
Unexpected behaviour of `camelCase`

### ISSUE THREAD

**Issue by djbb7** [lodash/lodash#5750]
The `camelCase` method returns some unexpected results when the provided text contains multiple uppercase characters in a single word. This is caused by the implementation of the `words` utility. 

```javascript
_.camelCase('IDs');
// returns => 'iDs'
// expected: 'ids'

_.camelCase('Product XMLs')
// returns => 'productXmLs'
// expected: 'productXmls'
```

**Comment by petertn** [comment]
Looks like this pull request will resolve this issue when approved

https://github.com/lodash/lodash/pull/5748

**Comment by jinbekim** [comment]
how #5748 can resolve this issue? I don't understand. 
This pr is just handling Pascalcase files.
> petertn

**Comment by petertn** [comment]
> how #5748 can resolve this issue? I don't understand. This pr is just handling Pascalcase files.
> 
> > Looks like this pull request will resolve this issue when approved #5748

You're right. I misunderstood the changes in those files based on the title.

**Comment by jhaprabhakar1998** [comment]
Hi @jinbekim,
Can I work on to fix the root cause of the issue. You have already added testcases, so if we are planning on fixing it, please let me know and I can pick it up.

**Comment by jhaprabhakar1998** [comment]
Hi @jdalton,
Thank you for creating this beautiful libray (making a lot life easier). Just wanted to check if you are working on releasing new versions/bug fixes (haven't seen new release from a long time). Also will be interested in working on this issue

**Comment by jinbekim** [comment]
Hi @jhaprabhakar1998 Sure, I'm not working on that now. Sorry for late reply

**Comment by jdalton** [comment]
Related to: #5836 — both cover `_.camelCase`/`_.kebabCase` handling of digits and casing boundaries.
